monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -87,10 +120,41 @@ class ConfigurePageComponent extends AuthComponent {
     if (this.state.selectedSection === 'attack') {
       this.matrixSubmit()
     } else {
-      this.configSubmit()
+      this.attemptConfigSubmit()
     }
   };
 
+  canSafelySubmitConfig(config) {
+    return !this.unsafeOptionsSelected(config) || this.state.unsafeOptionsConfirmed;
```

Comment:
Why do we need `this.state.unsafeOptionsConfirmed`

Reply:
This is a good question. Initially it was needed for the modal to work properly. This question prompted me to take a closer look and I was able to rework the logic and remove it.